ajax/libs/goalProgress/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "goalProgress",
+  "filename": "goalProgress.min.js",
+  "version": "1.0",
+  "description": "Animated progress bar using jQuery to show how close you are to reaching your goal.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tinacious/goalProgress.git",
+    "homepage": "http://tinacious.github.io/goalProgress/"
```

Comment:
Sorry for not mention detail.
Let's move `"homepage": "http://tinacious.github.io/goalProgress/"` between current line 10 and line 11.
Like
```js
  "repository": {
    "type": "git",
    "url": "https://github.com/tinacious/goalProgress.git",
  },
  "homepage": "http://tinacious.github.io/goalProgress/",
  "keywords": [
    "goalProgress",
    "tags",
    "jQuery",
    "animated"
  ],
```

Thank you.